packages/api/src/routes/conditionGroups.ts

```diff
@@ -0,0 +1,278 @@
+import { Request, Response, Router } from 'express';
+import prisma from '../db';
+
+const router = Router();
+
+// GET /admin/conditionGroups - list all groups with their conditions
+router.get('/', async (_req: Request, res: Response) => {
+  try {
+    const groups = await prisma.conditionGroup.findMany({
+      include: {
+        category: true,
+        condition: {
+          orderBy: { displayOrder: 'asc' },
+        },
+      },
+      orderBy: { createdAt: 'desc' },
+    });
+    return res.json(groups);
+  } catch (error: unknown) {
+    console.error('Error fetching condition groups:', error);
+    return res.status(500).json({ message: 'Internal Server Error' });
+  }
+});
+
+// POST /admin/conditionGroups - create a group
+router.post('/', async (req: Request, res: Response) => {
+  try {
+    const { name, categoryId, categorySlug } = req.body as {
+      name?: string;
+      categoryId?: number;
+      categorySlug?: string;
+    };
+
+    if (!name || !name.trim()) {
+      return res.status(400).json({ message: 'Name is required' });
+    }
+
+    let resolvedCategoryId: number | null = null;
+    if (typeof categoryId === 'number') {
+      resolvedCategoryId = categoryId;
+    } else if (categorySlug) {
+      const category = await prisma.category.findFirst({
+        where: { slug: categorySlug },
+      });
+      if (!category) {
+        return res
+          .status(404)
+          .json({ message: `Category with slug ${categorySlug} not found` });
+      }
+      resolvedCategoryId = category.id;
+    }
```

Comment:
add validation that either ``categoryId`` or ``categorySlug`` should be defined / resolvedCategoryId shouldn't be undefined by the time we create a group